docs/src/main/tut/guidelines.md

```diff
@@ -80,7 +80,7 @@ therefore name our implicits according to the following rules:
 - If the instance is for multiple type classes, use `InstancesFor` instead of a type class name.
 - If the instance is for a standard library type add `Std` after the package. i.e. `catsStdShowForVector` and `catsKernelStdGroupForTuple`.
 
-As an example, an implicit instance of `Monoid` for `List` defined in the package `Kernel` should be named `catsKernelMonoidForList`.
+As an example, an implicit instance of `Monoid` for `List` defined in the package `Kernel` should be named `catsKernelStdMonoidForList`.
```

Comment:
I am not sure if the renaming of these `catskernelXXXX` instances are worth the extra code due to BC 
The `std` in implicit names in `cats-core` was due to the fact that `cats.instances` package was used to be `cats.std`. And instances in `cats.kernel.instances`  don't have the `instance` package name in their names because having instances in an instance name is kind of redundant.  We have this long name rule in place simply so that contributors can easily come up with long unique name for these implicit defs to avoid potential conflict with other libraries. I agree that ideally we should keep it consistent but I think we are paying too much a cost for this kind of rename fix.